themes/default.theme/input.sass

```diff
@@ -4,3 +4,42 @@ input
     border
       color: $divider-color
       width: 1px
+  keyInput
+    background: white
+    inactive
+      margins: 40
+      color: "black"
+      background: $divider-color
+      animation
+        marginAnimation
+          duration: 300
+        colorAnimation
+          duration: 600
+    active
+      margins: 20
+      color: "black"
```

Comment:
Someone else actually requested black :smile:

But that's the point of themes! We need more anyways, if you have time to spare, you could start a new theme!